tests/cron.test.js

```diff
@@ -834,6 +867,42 @@ describe('cron', () => {
 			expect(callback).toHaveBeenCalledTimes(1);
 		});
 
+		it('should run a job using cron syntax with string format utcOffset with minute', () => {
+			const clock = sinon.useFakeTimers();
+			const callback = jest.fn();
+			const luxon = require('luxon');
+			// Current time
+			const t = luxon.DateTime.local();
+			// UTC Offset decreased by an 45 minutes (string format '(+/-)HH:mm')
+			const utcOffset = t.offset - 45;
+			let utcOffsetString = utcOffset > 0 ? '+' : '-';
```

Comment:
please use the same lines formatting [as your other test](https://github.com/kelektiv/node-cron/pull/685/files#diff-839e01ecdddc69537435c7ad3d7b1a2ce11a1d41644ada6836cb966c48a38519R803-R811) (also "an" in comment needs to be removed)

Reply:
thoughts on building a string with template literals instead of mutating a variable? like
```
const utcOffsetString = `
    ${utcOffset > 0 ? '+' : '-'}
    ${('0' + Math.floor(Math.abs(utcOffset) / 60)).slice(-2)}:
    ${('0' + Math.floor(Math.abs(utcOffset) / 60)).slice(-2)}
`;
 ```

Reply:
that's indeed more readable